package org.example.task.exception;public class UserAlreadyExistsException {
}
